modules/shortcodes/quiz.php

```diff
@@ -70,16 +70,35 @@ public static function init() {
 		add_shortcode( 'answer', array( __CLASS__, 'answer_shortcode' ) );
 		add_shortcode( 'wrong', array( __CLASS__, 'wrong_shortcode' ) );
 		add_shortcode( 'explanation', array( __CLASS__, 'explanation_shortcode' ) );
+
+		add_action( 'enqueue_block_editor_assets', array( __CLASS__, 'block_editor_assets' ) );
+		add_action( 'enqueue_block_assets', array( __CLASS__, 'enqueue_scripts' ) );
 	}
 
 	/**
 	 * Enqueue assets needed by the quiz,
 	 *
 	 * @since 4.5.0
 	 */
-	private static function enqueue_scripts() {
-		wp_enqueue_style( 'quiz', plugins_url( 'css/quiz.css', __FILE__ ) );
-		wp_enqueue_script( 'quiz', plugins_url( 'js/quiz.js', __FILE__ ), array( 'jquery' ), null, true );
+	public static function enqueue_scripts() {
+		if ( ! is_admin() ) {
+			wp_enqueue_style( 'quiz', plugins_url( 'css/quiz.css', __FILE__ ) );
```

Comment:
Should we flag this as rtl friendly via `wp_style_add_data( 'quiz', 'rtl', 'replace' );`